modules/editing-the-dashboard-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="editing-the-dashboard-configuration_{context}"]
+= Editing the dashboard configuration
+
+[role='_abstract']
+As an administrator, you can customize the interface of the dashboard by editing the dashboard configuration.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+
+.Procedure
+. Log in to the {openshift-platform} console as a cluster administrator.
+. In the *Administrator* perspective, click *Home* -> *API Explorer*.
+. In the search bar, enter `OdhDashboardConfig` to filter by kind.
+. Click the `OdhDashboardConfig` custom resource (CR) to open the resource details page.
+. Select the `redhat-ods-applications` project from the *Project* list.
```

Comment:
FYI, this namespace _can_ change now in the DSCI.

Reply:
@andrewballantyne Thank you for your feedback. As discussed in DM, I will make these updates in the next release, given the wider implications for the doc.